# HG changeset patch
# User John Rigby <john.rigby@vivint.com>
# Date 1430368108 21600
#      Wed Apr 29 22:28:28 2015 -0600
# Branch slimline
# Node ID 30e7ca64d1d65f79e2cc982c4d232be907f7ebad
# Parent  d98f9f5ceb743b6226019366ce2881f2322ffe09
fcc-hack fix qml errors and add wifi card dis/en-able

diff -r d98f9f5ceb74 -r 30e7ca64d1d6 code/roubaix/views/settings/SettingsFCCView.qml
--- a/code/roubaix/views/settings/SettingsFCCView.qml	Wed Apr 29 17:26:53 2015 -0600
+++ b/code/roubaix/views/settings/SettingsFCCView.qml	Wed Apr 29 22:28:28 2015 -0600
@@ -110,10 +110,22 @@
 
             RoubaixButtons.PropertyButton {
                 colorScheme: "orange"
-                label: qsTr("WPS Start)")
+                label: qsTr("WPS Start -- NOT YET WORKING")
                 onClicked: gController.system('wps.sh')
             }
 
+            RoubaixButtons.PropertyButton {
+                colorScheme: "orange"
+                label: qsTr("Disable wifi card")
+                onClicked: gController.system('echo 1 > /sys/bus/pci/devices/0000:01:00.0/remove')
+            }
+
+            RoubaixButtons.PropertyButton {
+                colorScheme: "orange"
+                label: qsTr("Enable wifi card")
+                onClicked: gController.system('echo 1 > /sys/bus/pci/devices/0000:00:00.0/rescan')
+            }
+
             Rectangle{
                 width:parent.width
                 height:labelHeight
@@ -176,52 +188,55 @@
             RoubaixButtons.PropertyButton {
                 colorScheme: "orange"
                 label: qsTr("slow")
-                onClicked: gController.system('fcc-helper led-slow')
+                onClicked: gController.system('fcc-helper led_slow')
             }
 
             RoubaixButtons.PropertyButton {
                 colorScheme: "orange"
                 label: qsTr("medium")
-                onClicked: gController.system('fcc-helper led-medium')
+                onClicked: gController.system('fcc-helper led_medium')
             }
 
             RoubaixButtons.PropertyButton {
                 colorScheme: "orange"
                 label: qsTr("fast")
-                onClicked: gController.system('fcc-helper led-fast')
+                onClicked: gController.system('fcc-helper led_fast')
             }
 
             RoubaixButtons.PropertyButton {
                 colorScheme: "orange"
                 label: qsTr("green++")
-                onClicked: gController.system('fcc-helper led-green-up')
+                onClicked: gController.system('fcc-helper led_green_up')
+            }
 
             RoubaixButtons.PropertyButton {
                 colorScheme: "orange"
                 label: qsTr("green--")
-                onClicked: gController.system('fcc-helper led-green-down')
+                onClicked: gController.system('fcc-helper led_green_down')
             }
 
             RoubaixButtons.PropertyButton {
                 colorScheme: "orange"
                 label: qsTr("red++")
-                onClicked: gController.system('fcc-helper led-red-up')
+                onClicked: gController.system('fcc-helper led_red_up')
+	    }
 
             RoubaixButtons.PropertyButton {
                 colorScheme: "orange"
                 label: qsTr("red--")
-                onClicked: gController.system('fcc-helper led-red-down')
+                onClicked: gController.system('fcc-helper led_red_down')
             }
 
             RoubaixButtons.PropertyButton {
                 colorScheme: "orange"
                 label: qsTr("blue++")
-                onClicked: gController.system('fcc-helper led-blue-up')
+                onClicked: gController.system('fcc-helper led_blue_up')
+            }
 
             RoubaixButtons.PropertyButton {
                 colorScheme: "orange"
                 label: qsTr("blue--")
-                onClicked: gController.system('fcc-helper led-blue-down')
+                onClicked: gController.system('fcc-helper led_blue_down')
             }
 
             Rectangle{
@@ -446,12 +461,12 @@
                 color:"transparent"
 
             }
+
             RoubaixButtons.PropertyButton {
                 colorScheme: "orange"
                 label: qsTr("Reboot Panel")
 		onClicked: gController.system('reboot')
             }
-
         }
     }
 }
